core/services/workflows/cmd/cre/utils/runner.go

```diff
@@ -16,31 +17,33 @@ type Runner struct {
 }
 
 type RunnerConfig struct {
-	enableBeholder             bool
-	enableBilling              bool
-	enableStandardCapabilities bool
-	lggr                       logger.Logger
-	engineHooks                v2.LifecycleHooks
+	EnableBeholder             bool
+	EnableBilling              bool
+	EnableStandardCapabilities bool
+	Lggr                       logger.Logger
+	LifecycleHooks             v2.LifecycleHooks
 }
 
 type RunnerHooks struct {
-	Initialize func(context.Context, RunnerConfig) (*capabilities.Registry, []services.Service)
-	BeforeRun  func(context.Context, RunnerConfig, *capabilities.Registry, []services.Service)
-	AfterRun   func(context.Context, RunnerConfig, *capabilities.Registry, []services.Service)
-	Cleanup    func(context.Context, RunnerConfig, *capabilities.Registry, []services.Service)
-	Finally    func(context.Context, RunnerConfig, *capabilities.Registry, []services.Service)
+	Initialize  func(context.Context, RunnerConfig) (*capabilities.Registry, []services.Service)
+	BeforeStart func(context.Context, RunnerConfig, *capabilities.Registry, []services.Service, []*pb.TriggerSubscription)
+	Wait        func(context.Context, RunnerConfig, *capabilities.Registry, []services.Service)
```

Comment:
Adding some doc string

```suggestion
        // Wait hook handles blocking for the runner to keep the standalone engine running
	Wait        func(context.Context, RunnerConfig, *capabilities.Registry, []services.Service)
```